scripts/msmarco/retrieve.py

```diff
@@ -12,12 +12,15 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+#
 
+import sys
 import argparse
 import time
-import os, sys
 
-from pyserini.search import pysearch
+sys.path.insert(0, './')
```

Comment:
Do we still need this?

Reply:
Yes, if user doesn't have `pip install`. I want it to work with both PyPI and dev installation.

Reply:
Got it, thanks!